cmake/linux/CMakeLists.txt

```diff
@@ -2,3 +2,9 @@ INSTALL(FILES lmms.png                  DESTINATION "${DATA_DIR}/pixmaps")
 INSTALL(FILES lmms                      DESTINATION "${DATA_DIR}/menu")
 INSTALL(FILES lmms.desktop              DESTINATION "${DATA_DIR}/applications")
 INSTALL(FILES lmms.xml                  DESTINATION "${DATA_DIR}/mime/packages")
+
+SET(LINUX_DESKTOP_FILE_EXEC "env QT_X11_NO_NATIVE_MENUBAR=1 lmms %f")
+SET(MENU_FILE_ICON_XPM "/usr/share/pixmaps/lmms.xpm")
```

Comment:
Is the menu something that was forgotten when removing the debian stuff or is this good for all Linuxes?  If we keep it, I would recommend simply `MENU_ICON` unless there is another specification besides XPM that is supported in the menu .   I read the freedesktop.org spec and this file doesn't seem to match, so please advise. Last, please group both `LINUX_DESKTOP_FOO` lines adjacent to each-other for organizational purposes.